user_modules/sequences/actions.lua

```diff
@@ -955,18 +931,16 @@ end
 --! @parameters: none
 --! @return: path to file
 --]]
-function m.sdl.getPTSFilePath()
-  return m.sdl.getSDLIniParameter("SystemFilesPath") .. "/" .. m.sdl.getSDLIniParameter("PathToSnapshot")
-end
+m.sdl.getPTSFilePath = SDL.PTS.file
 
 --[[ @sdl.getPTS: get Policy Table Snapshot (PTS)
 --! @parameters: none
 --! @return: table with PTS
 --]]
 function m.sdl.getPTS()
-  local ptsFileName = m.sdl.getPTSFilePath()
-  if utils.isFileExist(ptsFileName) then
-    pts = utils.jsonFileToTable(ptsFileName)
+  local remotePts = SDL.PTS.get()
+  if remotePts then
+    pts = remotePts
   end
```

Comment:
What's the difference between this and just `return SDL.PTS.get()`?

Reply:
@jacobkeeler The reason of this wrapper that we have to support 3 policy modes. 
As for `PROPRIETARY` or `EXTERNAL_PROPRIETARY` policy table  snapshot is a file on SDL host that could be obtained using `SDL.PTS.get()` function. However for `HTTP` mode snapshot is received in payload of `OnSystemRequest` (see `ptu.expectStart()` function)